packages/harmony/src/components/input/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,213 @@
+import { forwardRef, type ComponentPropsWithoutRef } from 'react'
+
+import cn from 'classnames'
+
+import layoutStyles from 'components/layout/layout.module.css'
```

Comment:
don't think layoutStyles exists in harmony anymore

Reply:
Yeah I started this PR before the layout stuff. Yoinked this in favor of the layout stuff of Flex components and layout tokens